cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -160,12 +166,12 @@ func (c *CStorPoolController) cStorPoolAddEvent(cStorPoolGot *apis.CStorPool) (s
 		pool.CStorZpools = map[string]zpool.Topology{}
 	}
 
-	// CheckValidPool is to check if pool attributes are correct.
 	devIDList, err := c.getDeviceIDs(cStorPoolGot)
 	if err != nil {
 		return string(apis.CStorPoolStatusOffline), errors.Wrapf(err, "failed to get device id of disks for csp %s", cStorPoolGot.Name)
 	}
-	err = pool.CheckValidPool(cStorPoolGot, devIDList)
+	// ValidatePool is to check if pool attributes are correct.
+	err = pool.ValidatePool(cStorPoolGot, devIDList)
```

Comment:
n_p: why the need to change the function name?
Verb + Adj + Noun ( e.g. getBadDisk(), validatePools(), removeOfflinePools() ) is followed IMO for general functions/methods like this.